schema/property.go

```diff
@@ -199,6 +209,10 @@ func parseItems(itemsRaw interface{}) *Property {
 	}
 }
 
+func parseEnum(enumRaw interface{}) []interface{} {
+	return enumRaw.([]interface{})
```

Comment:
Is it always an array? What if someone specifies single value for enum? I don't know if it's possible, a test would be great.

Reply:
enum is required to be an [array](https://github.com/cloudwan/gohan/blob/master/etc/schema/gohan.json#L197), nevertheless I'll add a test because AFAIR validation of nested properties may be broken in some places